engines/ngi/statesaver.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "ngi/gameloader.h"
 #include "ngi/objects.h"
+#include "engines/metaengine.h"
```

Comment:
Please move all non-engine includesbefore engine-specific ones. E.g. first go all the rest, then followed by `ngi/` includes. Please do it for all the modified files.